src/components/clusterConfiguration/DiscoveryInstructions.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import { Text } from '@patternfly/react-core';
 
-const DiscoveryInstructions: React.FC = () => (
+const DiscoveryInstructions = ({ isMultiNodeCluster = false }: { isMultiNodeCluster: boolean }) => (
```

Comment:
nit: I wold stay consistent for the sake of potential `git grep` in the future, so please rename to `isSingleNodeCluster`.

Reply:
Can do @mareklibra .

Could you please elaborate on which kind of consistency problems do you see and how they relate to `git grep`?

Reply:
We use `*SingleNode*` in similar cases, so I prefer to flip predicates instead of the naming.

When sustaining the codebase in the future (who knows by whom), it's common practice to use `git grep` for getting oriented or finding the places to fix. In case it is not an additional complication today, let's not make the task more difficult.